opam-ci-check/test/lint.t

```diff
@@ -57,44 +59,49 @@ Test the following:
 - [b.0.0.6] has a incorrectly formatted opam file
 - [b.0.0.7] has opam lint errors/warnings
 
-  $ opam-ci-check lint -r . -c b.0.0.1
+  $ opam-ci-check lint -r . b.0.0.1:new=false
```

Comment:
Is there test to cover the automatic detection of whether package is new?

Reply:
The name collision test now also tests the inference